main.go

```diff
@@ -25,6 +27,25 @@ type CmdLineOptions struct {
 	Validate   bool   `long:"validate" description:"Validate configuration and exit"`
 }
 
+var (
+	BuildID string
+	VERSION string
```

Comment:
style nits: looks like this `VERSION` variable isn't used, and `BuildID` can be a local variable inside `init()` below